src/lib.rs

```diff
@@ -291,6 +291,48 @@ impl<'c> CanvasElement<'c> {
       .compute()
   }
 
+  #[napi(js_name = "toBlob")]
+  pub fn to_blob(
+    &self,
+    env: Env,
+    callback: Function,
+    mime: Option<String>,
+    quality_or_config: Either3<f64, AvifConfig, Unknown>,
```

Comment:
according to the html spec, the quality should only be `0-1` `f64` here: https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/toBlob